src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -403,3 +403,11 @@ export const ADDITIONAL_EXPOSURES = Object.freeze({
   none: 'None of these',
   notsure: 'I’m not sure if I have been exposed to these hazards',
 });
+
```

Comment:
We may want to decide on a different way to organize constants as we keep adding content - like a new file for `/all-claims/constants/form0781` - but this is good for now.

Reply:
Valid suggestion.